src/styles/global.css

```diff
@@ -14,3 +14,43 @@ body {
 .ql-editor p em {
   font-style: italic;
 }
+
+strong {
```

Comment:
huh, but it's not what I've asked to do. Basically, what you did, is override of reset.css, which is probably fine, but hacky. Did you try to pass custom components to quill?

Reply:
I decided that was very bad if standard tag lost their meaning and return important part of their style for all leleka. ok i will check if i can change just for quill

Reply:
and we can change style not only for quill. Quill only help styling editor block but all other tweets rendered without quill and all this style its had to have too.